spec/cypress/app_commands/scenarios/ect_ucl.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+cip = FactoryBot.create(:core_induction_programme, name: "UCL")
+# You have to create this user in your spec before running this scenario
+User.find("53960d7f-1308-4de1-a56d-de03ea8e1d9a").core_induction_programme = cip
```

Comment:
I do wonder if that's the correct approach. 

I am quite tempted to load our seed file into our test database and use the users and programmes we have in there. It should make our tests reflect the dev apps more closely, and it ought to make the repeatable CIP setup less frustrating (and closer to reality). 

The cost obviously being the time it takes to load our seed file.

Do you want to see whether that idea makes sense in reality?

Reply:
There's two problems here:

- The way this is done now is a bit weird because the user was created in Cypress and then the scenario was ran afterwards - if we had the ability to log in a user created in a scenario we could merge all four of these scenarios into one and it would be a lot cleaner
- Yea, the lack of real data isn't ideal. We could give it a go, but I suspect a hybrid approach would be ideal - could we have a scenario which just loads the seed data?

Reply:
I'm happy for loading of seed data to be in a scenario if it makes our lives easier.